prod/index.js

```diff
@@ -591,24 +591,22 @@ module.exports.shareProject = function (token_filename,project_id,permission,use
 }
 
 module.exports.uploadCuratedPeakDataToCloud = function (signed_url, filePath) {
-    var options = {
-        method: 'PUT',
-        url: signed_url,
-        headers:
-            {
-                'x-amz-acl': 'bucket-owner-full-control',                
-            },
-        body: fs.readFileSync(filePath)
-    };
 
-    request(options, function (error, response, body) {
+    console.log(signed_url)
```

Comment:
Logging in console is not super-safe here since that is how we are communicating between the JS and C++ code. Please remove this unless it's being read by El-MAVEN